Makefile

```diff
@@ -198,7 +198,7 @@ shell/public/%.png: icons/%.svg
 shell/public/%-m.png: icons/%.svg
 	@convert -background none -scale 32x32 $< $@
 
-shell-build: shell/lib/* shell/client/* shell/server/* shell/shared/* shell/public/* shell/.meteor/packages shell/.meteor/release shell/.meteor/versions tmp/.shell-env
+shell-build: shell/lib/* shell/client/* shell/server/* shell/shared/* shell/public/* shell/.meteor/packages shell/.meteor/release shell/.meteor/versions tmp/.shell-env meteor-bundle-main.js
```

Comment:
`meteor-bundle-main.js` should now be a dependency of `bundle`, I think, since it's used by `make-bundle.sh`.